teams.gd

```diff
@@ -58,25 +57,21 @@ func select(team: String):
 			item_list.select(item,true)
 
 func _on_add_team_pressed():
-	add_team(text,true)
+	add_team($Text.text,true)
```

Comment:
using `@export var text: LineEdit` seems to be returning an error, since if I use `text.text`, then Godot returns an exception `Invalid get index 'text' (on base: 'Nil').`, and when using just `text`, the parser returns `Parser Error: Invalid argument for "add_team()" function: argument 1 should be "String" but is "LineEdit".`

it definitely must be me doing something wrong here, since tje same method with the ItemList appears to work just fine

Reply:
you forgot to set it in the inspector (select the teams panel and set it to the teams textbox)

Reply:
I see, thanks